configure

```diff
@@ -343,18 +343,31 @@ def b(value):
 
 
 def pkg_config(pkg):
-  cmd = os.popen('pkg-config --libs %s' % pkg, 'r')
-  libs = cmd.readline().strip()
-  ret = cmd.close()
-  if (ret): return None
+  pkg_config = os.environ.get('PKG_CONFIG', 'pkg-config')
+  pkg_config_args = '--silence-errors'
+  try:
+    subprocess.check_output([pkg_config, '--version'])
```

Comment:
My thought was to avoid one call if pkg-config wasn't installed. I'll have a go at improving this.